python/usepyhf/util.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+
+def create_json(ana_name:str, prefix:str):
+    """
+    Creates a JSON file in the /json folder with the name {ana_name}_{prefix}.json.
+    https://pyhf.readthedocs.io/en/v0.7.2/babel.html
+    """
+    if not os.path.isdir(f"./json/{ana_name}"):
+        os.mkdir(f"./json/{ana_name}")
```

Comment:
crashing here, no `make_parents` enabled. Also using `os` is discouraged in favor of `pathlib`.

Reply:
Strange, I have added a check which makes sure there is a /json folder in the first place, or creates one if there is none.
os.mkdir is already used in HistFitter.py, should we change all of it then?